app-utils/zellij/spec

```diff
@@ -2,3 +2,4 @@ VER=0.42.1
 SRCS="git::commit=tags/v${VER}::https://github.com/zellij-org/zellij"
 CHKSUMS="SKIP"
 CHKUPDATE="anitya::id=300087"
+REL=1
```

Comment:
Why bump REL here without any significant change?

Reply:
Dropped that commit. Changes were merged by git-rebase when 053576c4a8c850fb4b62a22e5c0802a5c3251728 landed in stable.